pkgs/applications/video/jellyfin-mpv-shim/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub
+, mpv, python-mpv-jsonipc, jellyfin-apiclient-python }:
+
+buildPythonApplication rec {
+  pname = "jellyfin-mpv-shim";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "iwalton3";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0bii85b08fa6sj57x9l8z5rhwlrz36mhf7nck950wzxfgkid2fdv";
+  };
+
+  # error: [Errno 13] Permission denied: '/homeless-shelter'
+  checkPhase = ''
+    export HOME=$TMPDIR
+  '';
+
+  propagatedBuildInputs = [ mpv python-mpv-jsonipc jellyfin-apiclient-python ];
```

please use 1 entry per line and order alphabetically. Also add pillow and tkinter, which are needed by executable
```suggestion
  propagatedBuildInputs = [
    jellyfin-apiclient-python
    mpv
    pillow
    python-mpv-jsonipc
    tkinter
  ];
```

still might not work right:
```
Traceback (most recent call last):
  File "/nix/store/fara2bdicvvi0p8bmrgcz6s8d6yicijc-python3-3.7.6/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/nix/store/p4jgrxxkkx7n5q92m32fpgc24p602r9k-jellyfin-mpv-shim-1.4.1/lib/python3.7/site-packages/jellyfin_mpv_shim/gui_mgr.py", line 380, in run
    from pystray import Icon, MenuItem, Menu
ModuleNotFoundError: No module named 'pystray'
```

---

Added and reordered the buildInputs.

I based this one on the plex-mpv-shim (#77884) which is also missing the GUI functionality:
```
[ WARNING] Cannot load GUI. Falling back to command line interface.
Traceback (most recent call last):
  File "/nix/store/faddcn3zaqj0q6zcanf6h6lglbp97dhm-plex-mpv-shim-1.7.12/lib/python3.7/site-packages/plex_mpv_shim/mpv_shim.py", line 39, in main
    from .gui_mgr import userInterface
  File "/nix/store/faddcn3zaqj0q6zcanf6h6lglbp97dhm-plex-mpv-shim-1.7.12/lib/python3.7/site-packages/plex_mpv_shim/gui_mgr.py", line 1, in <module>
    from pystray import Icon, MenuItem, Menu
ModuleNotFoundError: No module named 'pystray'
```

I will try to package the missing dependencies to enable the GUI.